sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from ..core.singleton import S
```

Comment:
Error is `ValueError: attempted relative import beyond top-level package`.
So I think this should work : 
`from sympy.core.singleton import S`